.github/workflows/build-lint-typecheck-test.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - DX-2714/fix-broken-ci-on-main
```

Comment:
do we need to permanently checking this in lint check?

Reply:
no :) Just pushed a commit that removes that line.